boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -900,6 +904,13 @@ pub unsafe fn start() -> (
     let scheduler = components::sched::round_robin::RoundRobinComponent::new(&PROCESSES)
         .finalize(components::round_robin_component_static!(NUM_PROCS));
 
+    let arm_dwt = static_init!(dwt::Dwt, dwt::Dwt::new());
+
+    let cycle_count = static_init!(
+        capsules_extra::cycle_count::CycleCount<dwt::Dwt>,
+        capsules_extra::cycle_count::CycleCount::new(arm_dwt)
+    );
+
```

Comment:
Do we want to add this to the nrf board? It seems too non-standard to me to add by default.

Reply:
nope, this is just there for testing // proof of compilation. I will remove before making this non-draft

Reply:
removed!